string localization_type
string object_type
---
nao_localization/ObjectMsg[] objects
bool success
